src/components/Nav.js

```diff
@@ -0,0 +1,61 @@
+// nav 레이아웃 구성
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import logoImg from "../assets/logo.svg";
+import './Nav.css';
+import Button from "./Button";
+import { getUser } from "../api";
+
+
+function Profile({userData}) {
+  console.log(userData);
+  // const email = userData.email;
+  // const image = userData.image;
+  // console.log(email);
+
+  return (
+    <div className="profile">
+      {/* <img className="profile-img" src={image} alt="프로필"/>
+      <span className="profile-email">{email}</span> */}
+    </div>
+  );
+}
+
+
+function Nav() {
+
+  // 프로필 데이터 가져오기 
+  const [userData, setUserData] = useState(null); 
+  const [isLoading, setIsLoading] = useState(false);
+  
+  const handleLoad = async () => {
+    try {
+      const response = await getUser();
```

Comment:
```suggestion
      const { data } = await getUser();
```
destruction 맘껏 활용해보세요!